src/biotite/structure/graphics/rna.py

```diff
@@ -0,0 +1,253 @@
+# This source code is part of the Biotite package and is distributed
+# under the 3-Clause BSD License. Please see 'LICENSE.rst' for further
+# information.
+
+__name__ = "biotite.structure.graphics"
+__author__ = "Tom David Müller"
+__all__ = ["plot_nucleotide_secondary_structure"]
+
+import shutil
+import numpy as np
+import matplotlib.pyplot as plt
+from .. import pseudoknots
+from ...application.viennarna import RNAplotApp
+
+
+def plot_nucleotide_secondary_structure(
+    base_labels, base_pairs, length, layout_type=1, draw_pseudoknots=True,
+    pseudoknot_order=None, angle=0, bond_linewidth=1, bond_linestyle=None, 
+    bond_color='black', backbone_linewidth=1, backbone_linestyle='solid', 
+    backbone_color='grey', base_font={'size': 'small'}, 
+    base_box={'pad': 0, 'color': 'white'}, annotation_positions=None, 
+    annotation_offset=8.5, annotation_font={'size': 'smaller'}, 
+    bin_path="RNAplot"
```

Comment:
As far as I know, it is not considered good practice to use mutable objects as default values for function parameters, in this case a `dict`. The reason is that accidental changes of these default values within the function still exist in subsequent function calls: https://florimond.dev/blog/articles/2018/08/python-mutable-defaults-are-the-source-of-all-evil/. A common alternative is using `None` as parameter and constructing the `dict` within the function.